ui/app/templates/components/job-page.hbs

```diff
@@ -22,7 +22,7 @@
       PlacementFailures=(component "job-page/parts/placement-failures" job=@job)
       TaskGroups=(component "job-page/parts/task-groups" job=@job)
       RecentAllocations=(component "job-page/parts/recent-allocations" job=@job activeTask=@activeTask setActiveTaskQueryParam=@setActiveTaskQueryParam)
-      Meta=(component "job-page/parts/meta" job=@job)
+      Meta=(component "job-page/parts/meta" meta=@job.meta)
```

Comment:
This. generally changes the context from "This is a component for jobs, it knows how to get metadata from a job" to "This is a component for metadata, it is given metadata"